includes/WeMail.php

```diff
@@ -294,6 +294,8 @@ private function includes() {
      */
     private function init_hooks() {
         register_activation_hook( WEMAIL_FILE, [ '\WeDevs\WeMail\Install', 'install' ] );
+        register_deactivation_hook( WEMAIL_FILE, [ '\WeDevs\WeMail\Uninstall', 'uninstall' ] );
+        add_action('admin_init', [ '\WeDevs\WeMail\Install', 'status_update' ], 20);
```

Comment:
Why do we need to update the status on each page load? What's the logic here?

Reply:
update: removed unnecessary action -   https://github.com/weMail/wemail/pull/107/commits/a0543587ca87c6f4fa0886de318fe2309093efd7